pkg/proxy/ipvs/netlink_linux.go

```diff
@@ -86,6 +86,16 @@ func (h *netlinkHandle) EnsureDummyDevice(devName string) (bool, error) {
 	return false, h.LinkAdd(dummy)
 }
 
+// IsDummyDeviceExist is part of interface
+func (h *netlinkHandle) IsDummyDeviceExist(devName string) bool {
```

Comment:
perhaps we should return `error` also?

Reply:
I think it is unnecessary. `IsDummyDeviceExist()` just checkout DummyDevice exist or not, focus much on `bool`, and we no need to return `err` generated on `IsDummyDeviceExist()`. Thank for your review.

Reply:
Great, thanks. done @m1093782566 @brendandburns